dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/internal/statemachine/StartAction.java

```diff
@@ -160,7 +165,8 @@ private void stopSlowStartMessage() {
     /** {@inheritDoc} */
     @Override
     public void cancel() {
-        _callback.set(null);
+        this.cancelled = true;
+        //      _callback.set(null);
```

Comment:
Did you intend to leave this commented out?  It is unclear to me if _callback will always get set from the CompletionListener when cancel is called and it is safe to remove this call.  But if that is all good then this commented out code will always make the next set of eyes have the same questions I do (I think).